luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -94,22 +94,25 @@ public LogicalTableDictionary getLogicalTableDictionary() {
 
     @Override
     public PhysicalTableDictionary getPhysicalTableDictionary() {
-        return resourceDictionaries.getPhysicalTableDictionary();
+        return resourceDictionaries.getPhysicalDictionary();
     }
 
     @Override
     public ResourceDictionaries getDictionaries() {
         return resourceDictionaries;
     }
 
-    public class Builder {
+    public static class Builder {
     
         private Map<String, Factory<Dimension>> dimensionFactories;
         private final ResourceDictionaries resourceDictionaries;
         public Builder(ResourceDictionaries resourceDictionaries) {
             this.resourceDictionaries = resourceDictionaries;
             dimensionFactories = getDefaultDimensionFactories();
         }
+        public Builder() {
```

Comment:
There should be an blank line between constructors:
```java
public Builder(ResourceDictionaries resourceDictionaries) {
...


public Builder() {
...

```

Reply:
Done.